recipes/coverage/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "3.7.1" %}
+{% set md5 = "c47b36ceb17eaff3ecfab3bcd347d0df" %}
+{% set pkgname = "coverage" %}
+
+package:
+  name: {{ pkgname }}
+  version: {{ version }}
+
+source:
+  fn: {{ pkgname }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ pkgname[0] }}/{{ pkgname }}/{{ pkgname }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
```

Comment:
Please see PR ( https://github.com/ericmjl/staged-recipes/pull/2 ) for removal of the versioned entrypoints. In the long run, I have asked upstream in issue ( https://github.com/nedbat/coveragepy/issues/13 ) if they can provide us an option to disable the installation of non-versioned entrypoints.